extensions/common/aws/aws-s3-core/src/main/java/org/eclipse/edc/aws/s3/S3BucketSchema.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2020, 2021 Microsoft Corporation
+ *  Copyright (c) 2023 Microsoft Corporation
```

Comment:
If you want to update the year on every source file you should not remove the "history", in this case it should be `2020 - 2023`, but feel free to not change them as well.
this comment is valid for all the other files where you changed the year